dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/view/impl/CreateNewPackageViewImpl.java

```diff
@@ -65,7 +61,8 @@ public CreateNewPackageViewImpl(Labels labels) {
         super(labels); 
         this.labels = labels;
         
-        getContinueButton().setText(labels.get(LabelKey.CONTINUE_BUTTON));
+        getContinueButton().setText(labels.get(LabelKey.NEXT_BUTTON));
```

Comment:
Oh no, just the constant name change for clarity. Now that I think about it, perhaps the button variable name and accessor should change too for consistency or we can just go back to the constant being CONTINUE.